src/main/java/ro/itschool/ema/models/dtos/EventDTO.java

```diff
@@ -0,0 +1,32 @@
+package ro.itschool.ema.models.dtos;
+
+import jakarta.validation.constraints.*;
+import lombok.Data;
+import org.springframework.validation.annotation.Validated;
+
+import java.io.Serializable;
+import java.util.Date;
+
+
+@Data
+@Validated
+public class EventDTO implements Serializable {
+    private Long id;
+    @NotNull(message = "Field cannot be null.")
+    private String eventName;
+
+    @NotNull(message = "Field cannot be null.")
+    @Future(message = "Invalid date.")
+    private Date eventDate;
+
+    @NotNull(message = "Field cannot be null.")
+    private String eventVenue;
+
+    @NotNull(message = "Field cannot be null.")
+    private String eventDescription;
+
+    @NotNull(message = "Field cannot be null.")
```

Comment:
an int variable cannot be Null. either remove the check or change the type to Integer